include/faabric/scheduler/Scheduler.h

```diff
@@ -80,6 +80,8 @@ class Scheduler
     std::string getThisHost();
 
     void broadcastFlush(const faabric::Message& msg);
+    
+    void flushLocalNodes(const faabric::Message& msg);
```

Comment:
We really need to change "nodes" to "executors" but it's used in too many places to do in this PR